lib/ocpp/v201/charge_point.cpp

```diff
@@ -264,6 +286,8 @@ void ChargePoint::on_transaction_finished(const int32_t evse_id, const DateTime&
                                 std::nullopt, std::nullopt, id_token_opt, meter_values, std::nullopt,
                                 this->is_offline(), std::nullopt);
 
+    this->database_handler->transaction_metervalues_clear(transaction_id);
```

Comment:
In case of a powerloss we might want to call this function at startup or clear the complete table in that case after the TransactionEvent.req(Ended) was sent. The complete handling for properly resuming/terminating transactions after power loss is propably something for another PR though.

Reply:
Currently this is done in the database_handler file itself. On init we clear the whole table. There should indeed be better handling of this but we don't have any context on that transaction anymore anyhow. So indeed something for a later PR.